lib/VisualGenerator.js

```diff
@@ -89,7 +89,104 @@ function copyVisualTemplate(targetPath, templateName) {
     fs.copySync(path.join(VISUAL_TEMPLATES_PATH, '_global'), targetPath);
     fs.copySync(path.join(VISUAL_TEMPLATES_PATH, templateName), targetPath);
 }
+/**
+ * Generate settings.ts
+ * 
+ * @param {string} targetPath - file path to root of visual package
+ * @param {string} verbose - to verbose output
+ */
+function generateSettings(targetPath, verbose) {
```

Comment:
There should be a UT that tests the VisualSettings object. 
- make sure the different members can be access (run)
- use templates/visuals/default/capabilities.json to get a good coverage
Since we're generating code we want to ensure it is 'runnable'. Test should run in pbiviz tools tests - not in the generated visual tests.